source/connect/mongoclient.txt

```diff
@@ -0,0 +1,120 @@
+.. _kotlin-sync-mongoclient:
+
+====================
+Create a MongoClient
+====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+To connect to a MongoDB deployment, you need two things:
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoClient** object, which creates the connection to and perform 
+  operations on the MongoDB deployment.
+
+You can also use either of these components to customize the way the {+driver-short+} behaves
```

Comment:
You can't really use MongoClient to customize.  You can use MongoClientSettings though.